create-snowpack-app/app-template-blank-typescript/public/index.html

```diff
@@ -6,6 +6,7 @@
     <meta name="viewport" content="width=device-width, initial-scale=1" />
     <meta name="description" content="Web site created using create-snowpack-app" />
     <link rel="stylesheet" type="text/css" href="/dist/index.css" />
+    <script type="module" src="/dist/index.js" defer></script>
```

Comment:
The "defer" attribute can be removed since modules defer automatically. Per https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Modules#other_differences_between_modules_and_standard_scripts

> There is no need to use the defer attribute when loading a module script; modules are deferred automatically.

Reply:
Good to know, but `@snowpack/plugin-webpack` removes `type="module"` from the final build so `defer` is still necessary here.